doc/book/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -0,0 +1,106 @@
+a laminas-mvc based application.
+The example creates a listener for logging errors of an application with [laminas-log](https://docs.laminas.dev/laminas-log/).
+
+Before starting, make sure laminas-log is [installed and configured](https://docs.laminas.dev/laminas-log/installation/).
+
+laminas-eventmanager is already present in laminas-mvc based applications as it is an event-driven MVC layer based on the event manager.
+
+## Create Listener
+
+Create a listener as class using [a listener aggregate](../aggregates.md) and add the logger via constructor injection to create a valid object with all required dependencies, e.g. `module/Application/src/Listener/ErrorListener.php`:
+
+```php
+namespace Application\Listener;
+
+use Exception;
+use Laminas\EventManager\AbstractListenerAggregate;
+use Laminas\EventManager\EventManagerInterface;
+use Laminas\Log\Logger;
+use Laminas\Mvc\MvcEvent;
+
+final class ErrorListener extends AbstractListenerAggregate
+{
+    public function __construct(
+        private readonly Logger $logger
+    ) {}
+
+    public function attach(EventManagerInterface $events, $priority = 1): void
+    {
+        $this->listeners[] = $events->attach(
+            MvcEvent::EVENT_DISPATCH_ERROR,
+            [$this, 'onDispatchError']
+        );
+    }
+
+    public function onDispatchError(MvcEvent $event): void
+    {
+        /** @var Exception|null $exception */
+        $exception = $event->getParam('exception');
+        if ($exception) {
+            $this->logger->crit('Error: ' . $exception->getMessage());
+        } else {
+            $this->logger->crit('Error: ' . $event->getError());
+        }
+    }
+}
+```
+
+The listener aggregate was chosen because [it can listen to even more events](../aggregates.md#recommendations) due to the possible extension.
+
+NOTE: **More Events**
+All events that can be triggered are listed and explained in the [documentation of laminas-mvc](https://docs.laminas.dev/laminas-mvc/mvc-event/).
+
+## Register Listener
+
+To register a listener in a laminas-mvc based application use the application or module configuration, such as `config/autload/global.php` or `module/Application/config/module.config.php`, and the configuration key `listeners`.
+
+The current example uses the module configuration, e.g. `module/Application/config/module.config.php`:
+
+<pre class="language-php" data-line="4-6"><code>
+namespace Application;
+
+return [
+    'listeners' => [
+        Listener\ErrorListener::class,
+    ],
+    // …
+];
+</code></pre>
+
+All listeners registered in this way are fetched from the application service container.
+This means the listeners must be registered for the application service container.
+
+To register the listener for the application service container, extend the configuration of the module.
+Add the following lines to the module configuration file, e.g. `module/Application/config/module.config.php`:
+
+<pre class="language-php" data-line="3,8"><code>
```

Comment:
This uses `data-line="3,8"`, while above it uses a range? Intentional? (just asking - unfamiliar with syntax in here)

Reply:
Lines can be highlighted as groups or individually, and there can be multiple highlights in one block of code.